### ISSUE TITLE
Simplify subjects and tags

### ISSUE THREAD

**Issue by abm-devmonk-purple** [abhayagiri/abhayagiri-website#97]
Tags were part of the early effort of organizing the talks. As later schemes were added, namely subjects and playlists, the role of tags were changed to being a bridge between subjects and talks.

However, the tags are never featured anywhere on the website. Further, they complicate data entry and management because it's hard to see the how adding a particular tag leads to a subject being added to a talk.

In short, we would like to:

1. Remove `/admin/tags`.
2. Add a field in `admin/talks` to add 0 or more subjects.
